Makefile

```diff
@@ -151,5 +151,8 @@ build-image-tnf:
 		-t ${REGISTRY}/${TNF_IMAGE_NAME}:${TNF_VERSION} \
 		-f Dockerfile .
 
+# Generates the classification.js file and creates anew result-embed.html
 classification-js:
 	./tnf generate catalog javascript > script/classification.js
+	sed '/<script src=".\/classification.js"><\/script>/e echo "<script>";cat  script/classification.js; echo "<\/script>"' script/results.html > script/results-embed.html
```

Comment:
Space is not needed after redirect `>/dev/null`. I think `shfmt` complains.

Reply:
fixed